### DISCUSSION TITLE
Docs: Light theme not applied in search box

### DISCUSSION THREAD

**Discussion by andrey-helldar** [laravel/framework#40983] [Ideas]
I'm using a light theme, but after updating the site, the search box is displayed in a dark window. My eyes hurt when reading text on a dark screen, so I almost stopped using the search, which makes it difficult to find a solution.

I propose to add a definition of a pre-installed light or dark theme in the browser and choose the setting for the search box as well.

![image](https://user-images.githubusercontent.com/10347617/153730607-ab64c94d-f1d0-40b9-8573-dc8777342979.png)

![image](https://user-images.githubusercontent.com/10347617/153730612-9d35fa0f-5baf-45f3-853d-4a47e336ed61.png)

**Comment by Sophist-UK** [comment]
See also https://github.com/laravel/framework/discussions/46425.

**Reply by andrey-helldar** [reply]
On March 17, the form started showing light mode and I closed the discussion. But thanks for the link 🤝

![image](https://user-images.githubusercontent.com/10347617/225961319-3ac6ba45-dea2-415e-83a1-af3875a754da.png)
